packages/react/src/profile/index.ts

```diff
@@ -11,6 +11,7 @@ export * from './useCollectedPublications';
 export * from './useMutualFollowers';
 export * from './useUpdateProfileImage';
 export * from './useProfilesOwnedBy';
+export * from '../publication/useWhoMirroredPublication';
```

Comment:
is this necessary? I think you just export it from the publication/index.ts, not from profile folder